### PR TITLE
Update pendulum.py

### PR DESCRIPTION
**Pull request by zeeshanalipanhwar**
I have added the description of the class as much as I could, it needs to be completed.

### CODE REVIEW

**Comment by jkterry1** [comment]
CAn you please make tests pass?

File: gym/envs/classic_control/pendulum.py [hunk 1]
```diff
@@ -6,6 +6,28 @@
...
+        The inverted pendulum swingup problem is a classic
+        problem in the control literature. In this version of the
+        problem, the pendulum starts in a random position, and the
+        goal is to swing it up so it stays upright.
+    Source:
+    Observation:
+        Type: Box(3)
+        Num     Observation               Min                     Max
+        0                                 -1                       1
+        1                                 -1                       1
```
Comment (RedTachyon) [inline gym/envs/classic_control/pendulum.py:20]: The first two observations are missing, what is their interpretation?

File: gym/envs/classic_control/pendulum.py [hunk 2]
```diff
@@ -6,6 +6,28 @@
...
+        2       Speed                     -8                       8
+    Actions:
+        Type: Box(2)
+        Num   Action
+        0     Apply torque -2
+        1     Apply torque +2
+    Reward:
+    Starting State:
+    Episode Termination:
```
Comment (RedTachyon) [inline gym/envs/classic_control/pendulum.py:29]: Reward, Starting State and Episode Termination are also missing, I'm pretty sure all of that can be easily found in the code.

File: gym/envs/classic_control/pendulum.py [hunk 3]
```diff
@@ -6,6 +6,28 @@
...
+        goal is to swing it up so it stays upright.
+    Source:
+    Observation:
+        Type: Box(3)
+        Num     Observation               Min                     Max
+        0                                 -1                       1
+        1                                 -1                       1
+        2       Speed                     -8                       8
+    Actions:
+        Type: Box(2)
```
Comment (RedTachyon) [inline gym/envs/classic_control/pendulum.py:23]: This is incorrect, the action space is a `Box(1)`, with the sole action applying a continuous torque in the range [-2, 2]

File: gym/envs/classic_control/pendulum.py (hunk 1 above)
Comment (zeeshanalipanhwar) [reply gym/envs/classic_control/pendulum.py:20]: Those are the Mass and the Length of the Pendulum.

**Comment by RedTachyon** [comment]
>Those are the Mass and the Length of the Pendulum.

This doesn't seem right, why would the observation be mass and length? Those are constants of the environment.

**Comment by RedTachyon** [comment]
@jkterry1 I'd suggest reverting this merge, or doing another PR which would do this correctly. My probelms with the current version:
- Observation space is specified as Box(1) while in reality it's Box(3)
- It's not well formatted, which means that tests will be failing for every PR branching from current master
- Reward isn't specified
- Starting State isn't specified
- Episode Termination isn't specified

Imo it's better to have no docstring compared to a docstring that's actively wrong. I can try to get a proper docstring together at some point, but especially for the time you're gone (if you're still around for now), I'd really prefer if this was reverted

**Comment by jkterry1** [comment]
@RedTachyon just created and merged a PR reverting. If you could create a PR that does this correctly while I'm on vacation that'd be great.

**Pull request merged**